cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js

```diff
@@ -91,6 +91,7 @@ function displayPaymentMethods() {
 function addPosTerminals(terminals) {
     //create dropdown and populate connected terminals
     var dd_terminals = document.createElement("select");
+    dd_terminals.id = "terminalList";
```

Comment:
why not adding this into the template ?

Reply:
Will add this dynamically based on terminals from /connectedTerminals call